apps/backend/src/libs/plugins/file-plugin/libs/enums/image-mimetype.enum.ts

```diff
@@ -0,0 +1,9 @@
+const ImageMimetype = {
+	BMP: "image/bmp",
+	GIF: "image/gif",
+	JPEG: "image/jpeg",
+	PNG: "image/png",
+	X_PNG: "image/x-png",
```

Comment:
Let's add these to existing ContentType enum. And we will allow only PNG and JPEG